update localcluster deployment doc to use Docker desktop for local deployment Resolves issue #3819 (#3820)
kubeflow/website: content/en/docs/components/pipelines/legacy-v1/installation/localcluster-deployment.md

For more information about K3ai, refer to the
[official documentation](https://k3ai.github.io/docs/intro).

## Docker-Desktop

Docker Desktop is secure, out-of-the-box containerization software offering developers and teams a robust, hybrid toolkit to build, share, and run applications anywhere.

> ### 1. installing Docker-Desktop

Reviewer: ```suggestion
### 1. Installing Docker-Desktop
```

You can install and configure Docker-Desktop by following the
[official quick start](https://www.docker.com/products/docker-desktop/).

**on Windows** - download and run the Docker Desktop Installer.exe file, and follow the instructions inside the installer.

### 2. Creating a cluster on Docker-Desktop

Having installed Docker-Desktop, you can create a Kubernetes cluster on Docker-Desktop by following those steps:

> 1. open the Docker-Desktop dashboard.

Reviewer: ```suggestion
1. Open the Docker-Desktop dashboard.
```

> 2. open the settings by pressing the settings button.

Reviewer: ```suggestion
2. Open the settings by pressing the settings button.
```

> 3. navigating to the Kubernetes tab on the left side.

Reviewer: ```suggestion
3. Navigate to the Kubernetes tab on the left side.
```

> 4. checking the 'Enable Kubernetes' checkbox, and pressing on 'Apply and restart'. and wait for the app to restart.

Reviewer: ```suggestion
4. Check the 'Enable Kubernetes' checkbox, click 'Apply and restart' and wait for the app to restart.
```

> 5. in order to check if the cluster is up and running, open the the command prompt and run the following command: "kubectl cluster-info" and you should see the following output:
>    "Kubernetes control plane is running at https://kubernetes.docker.internal:<some-port>
>    CoreDNS is running at https://kubernetes.docker.internal:<some-port>/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy

Reviewer: ```suggestion
5. In order to check if the cluster is up and running, open the command prompt and run the following command:
    ```SHELL
    kubectl cluster-info   
    ```
    You should see the following output:
    ```SHELL
    Kubernetes control plane is running at https://kubernetes.docker.internal:<some-port>
    CoreDNS is running at https://kubernetes.docker.internal:<some-port>/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
    ```
```

> To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.

Reviewer: ```suggestion
To further debug and diagnose cluster problems, use `kubectl cluster-info dump`.
```

Author: thanks for your comments, i took your advices and made the changes

For more information about K3ai, refer to the
[official documentation](https://k3ai.github.io/docs/intro).

## Docker-Desktop

Docker Desktop is secure, out-of-the-box containerization software offering developers and teams a robust, hybrid toolkit to build, share, and run applications anywhere.

### 1. Installing Docker-Desktop

You can install and configure Docker-Desktop by following the
[official quick start](https://www.docker.com/products/docker-desktop/).

**on Windows** - download and run the Docker Desktop Installer.exe file, and follow the instructions inside the installer.

### 2. Creating a cluster on Docker-Desktop

Having installed Docker-Desktop, you can create a Kubernetes cluster on Docker-Desktop by following those steps:
1. Open the Docker-Desktop dashboard.

> 2. Open the settings by pressing the settings button.

Reviewer: ```suggestion
2. Open the settings by clicking the settings button.
```

> 3. Navigating to the Kubernetes tab on the left side.

Reviewer: ```suggestion
3. Navigate to the Kubernetes tab on the left side.
```

> 4. Checking the 'Enable Kubernetes' checkbox, and pressing on 'Apply and restart'. and wait for the app to restart.

Reviewer: ```suggestion
4. Check the 'Enable Kubernetes' checkbox, click 'Apply and restart' and wait for the app to restart.
```

For more information about K3ai, refer to the
[official documentation](https://k3ai.github.io/docs/intro).

## Docker-Desktop

Docker Desktop is secure, out-of-the-box containerization software offering developers and teams a robust, hybrid toolkit to build, share, and run applications anywhere.

### 1. Installing Docker-Desktop

You can install and configure Docker-Desktop by following the
[official quick start](https://www.docker.com/products/docker-desktop/).

**on Windows** - 
download and run the Docker Desktop Installer.exe file, and follow the instructions inside the installer.

### 2. Creating a cluster on Docker-Desktop

Having installed Docker-Desktop, you can create a Kubernetes cluster on Docker-Desktop by following those steps:
1. Open the Docker-Desktop dashboard.
2. Open the settings by clicking the settings button.
3. Navigate to the Kubernetes tab on the left side.

> 4. check the 'Enable Kubernetes' checkbox, click on 'Apply and restart'. and wait for the app to restart.

Reviewer: ```suggestion
4. Check the 'Enable Kubernetes' checkbox, click 'Apply and restart' and wait for the app to restart.
```

> 4. Check the 'Enable Kubernetes' checkbox, click on 'Apply and restart'. and wait for the app to restart.

Author: i am sorry didn't noticed those.. fixed them now
